tests/test_read.py

```diff
@@ -100,25 +111,23 @@ def test_read_icc_color_profile():
 def test_read_pillow_frombytes():
     for fn in Path().glob("tests/images/**/*.heic"):
```

Comment:
I would move all `for fn in Path()....:` into test parametrization, so test is just `def test_read_pillow_frombytes(heic_fn):`, and globbing goes into either `conftest.pytest_generate_tests` or into decorator.

Reply:
https://github.com/uploadcare/pyheif/pull/4